package.json

```diff
@@ -44,6 +44,7 @@
     ],
     "activationEvents": [
         "onLanguage:python",
+        "onLanguage:markdown",
```

Comment:
Code looks pretty reasonable. But one more thing popping out here. The code expressions are added on activation via the lens manager. But it's only activating on python and markdown files loading. The languages that we add to codeExpressions would need to also register here as activation events. But that feels like it would have to be a dynamic list here. Not sure if that is possible.

Reply:
Something to ask VS code then. Can a user add activation events to their settings.json for an extension.